{{cookiecutter.project_name}}/{{cookiecutter.package_name}}/__init__.py

```diff
@@ -1 +1,18 @@
-"""{{cookiecutter.package_name}}."""
```

Comment:
issue (blocking): This comment line helps to keep the quality mettrics: linter's won't complain about the file not having a module docstring.

Reply:
OK, will fix. But this feels like a linter error to me :p